docs/IRIDANext_LocalGuide.md

```diff
@@ -0,0 +1,514 @@
+
+```bash
+createuser -s test -P -U postgres
+# When prompted for a password for the `test` role above, set the password as `test`. These are the credentials used by the development and test databases.
+```
+
+**8. Generate and configure the environment credentials**
+
+Refer to [GA4GH WES Sapporo Setup](#ga4gh-wes-sapporo-setup-for-development) for how to setup and configure the IRIDA Next environment credentials for development purposes.
+
+**9. Initialize the databases**
+
+_Run:_
+
+```bash
+cd /path/to/irida-next
+bin/rails db:create db:migrate db:seed
+```
+
+**10. Register pipelines**
+
+Refer to [Registering pipelines in IRIDA Next](#registering-pipelines-in-irida-next) for how to register Nextflow pipelines.
+
+**11. Restarting**
+
+Rebuild and restart the sapporo docker container, restart postgres, and start Irida Next following the steps found here: [Running a local instance of IRIDA Next](#running-a-local-instance-of-irida-next).
+
+# [GA4GH WES Sapporo Setup for development](https://github.com/phac-nml/irida-next/blob/main/docs-site/docs/development/integration/ga4gh_wes_sapporo_setup.md)
+
+## Prerequisites
+
+You will need to have a Docker Compose plugin installed to manage Sapporo.
+
+_Install Docker Compose plugin on Ubuntu_
+
+```bash
+sudo apt-get update
+sudo apt-get install docker-compose-plugin
+# Verify installation
+docker compose version
+# Should be version v2.26.1 or newer
+```
+
+_Add yourself to the docker group to run docker commands_
+
+```bash
+sudo groupadd docker
+sudo usermod -aG docker $USER
+# Reboot, log out/log in, or run the following command
+newgrp docker
+#If you still have issues with running docker, you may need to change the permissions of the docker socket using the following command
+sudo chmod 666 /var/run/docker.sock
+```
+
+## Configure IRIDA Next environment developer credentials
+
+_Check that your active storage service is set to `:local` in `config/environments/development.rb`_
+
+```bash
+# check if the active storage service is set to :local
+grep -q "config.active_storage.service = :local" config/environments/development.rb && echo "Active storage service is set to :local" || echo "Active storage service is not set to :local"
+```
+
+_Configure environment developer credentials_
+
+```bash
+EDITOR="vim --nofork" bin/rails credentials:edit --environment development
+```
+
+```bash
+# Add the following to the credentials file and save in the text editor
+ga4gh_wes:
+  server_url_endpoint: 'http://localhost:1122/'
+```
+
+- Make sure that there's no comment characters (`#`) preceding these lines!
+
+- You may replace `vim` with any text editor of your preference.
+
+## Setup Sapporo (WES implementation)
+
+_Download and run the [PHAC-NML Sapporo]() fork in dev docker mode. If docker permissions are setup correctly you should not have to use `sudo` when running these commands._
+
+```bash
+cd ~/path/to/git/repos
+git clone git@github.com:phac-nml/sapporo-service.git
```

Comment:
Could you change this to cloning the `https` git URL instead? Since git@github.com requires permissions on the repo.